packages/runtime/src/state/react-page.ts

```diff
@@ -262,43 +209,18 @@ export function copyElementTree(
   return copyElementTreeNode(state, createReplacementContext(replacementContext))(copy)
 }
 
-function* traverseElementTree(
-  state: State,
-  elementTree: Documents.ElementData,
-): Generator<Documents.Element> {
-  yield elementTree
```

Comment:
Moved to https://github.com/makeswift/makeswift/pull/894/files#diff-4c77c71de0cb4022cef56ea1671d57f758f297a229dab9ae8caf6f307c7b7918R91